app/controllers/candidates_controller.rb

```diff
@@ -0,0 +1,2 @@
+class CandidatesController < ApplicationController
```

Comment:
This isn't needed, as you are using `accepts_nested_attributes_for`, so you can remove this :-)